typescript/infra/scripts/check/check-validator-announce.ts

```diff
@@ -25,7 +25,7 @@ async function main() {
 
   const targetNetworks = (
     chains && chains.length > 0 ? chains : config.supportedChainNames
-  ).filter(isEthereumProtocolChain);
+  ).filter((chain) => isEthereumProtocolChain(chain) && chain !== 'lumia');
```

Comment:
why not lumia?

Reply:
I was thinking of working around and adding the lumia addresses to the metadata manually so the script still works, but thought it easier to skip lumia instead

can update the script if we want

Reply:
decided to just leave a comment for why not lumia